packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -67,6 +67,19 @@ export interface ProductGalleryProps {
   }
   sortBySelector?: SortProps
   productCard?: Pick<ProductCardProps, 'showDiscountBadge' | 'bordered'>
+  emptyGallery?: {
+    title?: string
+    topButton?: {
+      text: string
+      href: string
+      iconName: string
```

Comment:
```suggestion
      label: string
      url: string
      icon: string
```

Reply:
yes sure! Im pretty sure I changed it everywhere